Week2/homework/gradecalc.js

```diff
@@ -0,0 +1,32 @@
+"use strict"
+
+/* the function below is to turn the score into percentual*/
+function perc (score, total) {
+    return score * total / total
+}
+
+/*the function below will put the score into a class of grade*/
+function whichgrade (score) {
+    if (score >= 0 && score <= 49){
+        grade= "F"
```

Comment:
Ah. You missed declaring the variable `grade` here before defining it. I would recommend adding another if condition at the top to check if score is between 0 and 100 and is a valid number. Also, would suggest you check if you can remove the first part of && condition and see if the code still works as expected. 🙂